third_party/airflow/armada/logs/log_consumer.py

```diff
@@ -0,0 +1,252 @@
+# Copyright 2016-2024 The Apache Software Foundation
```

Comment:
We should add this license to all files or none of them.  @dave-gantenbein?

Reply:
Not to all files, that would be overly cumbersome!

Reply:
in which case can we get rid of the license header from these files